### ISSUE TITLE
Difference between information provided by /collections and /collections/collectionId

### ISSUE THREAD

**Issue by ghobona** [opengeospatial/OGC-API-Sprint-August-2020#9]
Currently there is an expectation that `/collections/collectionId` will return a more detailed response compared to `/collections`. However, it is unclear what exactly should be returned in `/collections`.

**Comment by cmheazel** [comment]
The August 10 update to the specification includes a short discussion on this topic.  There is also a place holder for the list of recommended (recommendation 1) elements for the /collections coverage information. I hope this sprint will provide me what I need to populate it.

**Comment by joanma747** [comment]
Let me reiterate that /collections/collectionId is data-model-less. This means that at this point you do not know if your data is available as items (features) or coverages or other future "thing". Any new element on this response should be neutral. Adding the resolution of the data (in scaleDenominator?) or a more multidimensional bbox are possibilities but not much more than that. 

Assuming that the attributes of a feature can be mapped to the values of a coverage (rangetype names), the idea of "queriables" (suggested in Testbed 15) could be used  and reused by features or coverages or... So that could be another addition.

**Comment by jerstlouis** [comment]
@joanma747 while the collection of data itself conceptually is data-model-less, a particular API can still offer details in it (extending Common - Part 2) which are speciic to a supported API (view, e.g. _Features_ and/or _Coverages_) way to access that data.

At least that was the conclusion of https://github.com/opengeospatial/ogc_api_coverages/issues/64 which I believe included back and forth with Common, and resulted in e.g. extending "extent" to add extra dimensions, and links or properties in `.../{collectionId}` to specify the DomainSet and RangeType at that level.

This allows a simple `/coverage` just like you have a simple `/items` for Features. This also has me thinking again about `/map` being a description resource rather than a default map already, especially that the content of `/map` currently looks a lot like `/styles` :)  (e.g. https://maps.ecere.com/ogcapi/collections/Daraa/styles vs. https://maps.ecere.com/ogcapi/collections/Daraa/map).

**Comment by joanma747** [comment]
Even if I sympathize with the idea of having model dependent information in .../collections/{collectionId} we avoided that in OGC API maps and tiles because we became convinced by others that it was not a good thing. I believe that the results is not so bad. I'll not start now populating the .../collections/{collectionId} with too many API specific info. There is nothing wrong with having this info directly in .../collections/{collectionId}/coverage

The alternative could be to allow everybody to include model dependent info in .../collections/{collectionId} and do that for performance reasons (e.g. reduce the number of http interactions.

So please, lets reach a consensus on that soon and let's try to be consistent in the different API standards.

**Comment by cmheazel** [comment]
One principle the JSON people have pounded into my head is that what is not recognized is ignored. So JSON, at least, is inherently extensible. However, "do your own thing" is not a good approach for a standard. So let's work out an approach for extending the `collection information` and then take that back to Common as a recommended mod.

**Comment by jerstlouis** [comment]
@joanma747 

> The alternative could be to allow everybody to include model dependent info in .../collections/{collectionId} and do that for performance reasons (e.g. reduce the number of http interactions.

That has my vote. Agreed it should be consistent.

> There is nothing wrong with having this info directly in .../collections/{collectionId}/coverage

What this allowed is for `/coverage` to directly be your GeoTIFF or netCDF files where you can also do your subsetting etc., rather than being a metadata doument like map right now.

Consequently this matches very nicely to Features where `/items` is directly your GeoJSON document.

The domainset and rangetype might be part of the coverage, but having this info consistently always available in CIS JSON ensures that clients will always have a clear standard CIS-based description of the range set and domainset in a consistent manner, even if the coverage as a whole is only available in e.g. GeoTIFF.

**Comment by Schpidi** [comment]
I agree with @jerstlouis, if we want `/coverage` to return the coverage itself e.g. as GeoTIFF than we need a higher path to provide the information needed to generate proper parameter values for the `/coverage` path, i.e. the range type and domain set.
